bot/cogs/loop.py

```diff
@@ -0,0 +1,40 @@
+import io
+from datetime import datetime, time
+
+import discord
+import pytz
+from discord.ext import commands, tasks
+
+from bot.bot import WarnetBot
+from bot.config import ADMIN_CHANNEL_ID, CustomRoleConfig, GUILD_ID
+
+
+class Loop(commands.Cog):
```

Comment:
Change the class name to avoid ambiguity from other libs. Perhaps `Booster` or something else similar.